etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -1,4 +1,33 @@
 ---
+- name: Pre-checks
+  gather_facts: false
+  hosts: localhost
+  tags:
+    - prechecks
+  tasks:
+    - name: Fail if using old path for Wazuh certificates
+      fail:
+        msg: >-
+          The path used for Wazuh SSL certificates was changed in a previous release. The certificates
+          were found in the wrong location. Please move them from {{ playbook_dir }}/wazuh/certificates
+          to {{ kayobe_env_config_path }}/wazuh/certificates, e.g git mv {{ playbook_dir }}/wazuh/certificates
+          {{ kayobe_env_config_path }}/wazuh/certificates
+      when: (playbook_dir ~ '/wazuh/certificates') is exists
+
+    - name: Fail if using old path for custom certificates
+      fail:
+        msg: >-
+          Wazuh custom SSL certificates have been merged with regular certificates. The certificates
+          were found in the wrong location. Please move them from {{ playbook_dir }}/wazuh/custom-certificates
+          to {{ kayobe_env_config_path }}/wazuh/certificates/certs ,e.g git mv {{ playbook_dir }}/wazuh/custom-certificates/*
+          {{ kayobe_env_config_path }}/wazuh/certificates/certs.
```

Comment:
Isn't it the other way around? Wazuh generates certs in the certs subdirectory, but custom certs just go in this directory?

Reply:
Hmm, This check is for the custom certs, so I was trying to move them into "certs"

Reply:
I think I see what you mean, In the first example I was moving the whole directory, whereas I was trying to move individual files in the second.